packages/create-cloudflare/src/common.ts

```diff
@@ -315,3 +315,21 @@ export async function initializeGit(cwd: string) {
 		await runCommand(`git init`, { useSpinner: false, silent: true, cwd });
 	}
 }
+
+export async function getProductionBranch(cwd: string) {
+	try {
+		const producitonBranch = await runCommand(
+			"git rev-parse --abbrev-ref HEAD",
```

Comment:
Is there a reason to not use `git branch --show-current`? If it's to support older version of git, I suggest:

```suggestion
      // "git branch --show-current", // git@^2.22
			"git rev-parse --abbrev-ref HEAD", // git@^1.6.3
```

Reply:
Nope no reason, I just copied the wrangler code. Good suggestion